alicloud/provider.go

```diff
@@ -141,6 +141,7 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_api_gateway_groups":       dataSourceAlicloudApiGatewayGroups(),
 			"alicloud_api_gateway_apps":         dataSourceAlicloudApiGatewayApps(),
 			"alicloud_elasticsearch_instances":  dataSourceAlicloudElasticsearch(),
+			"alicloud_cas_certs":                dataSourceAlicloudCasCerts(),
```

Comment:
Please use the whole datasource name `alicloud_cas_certificates`

Reply:
done